addon/components/frost-action-link.js

```diff
@@ -0,0 +1,30 @@
+import Ember from 'ember'
+import layout from '../templates/components/frost-action-link'
+import _frostObjectDetails from './frost-object-details'
+import _ from 'lodash/lodash'
+
+export default Ember.Component.extend({
+  routing: Ember.inject.service('-routing'),
+
+  layout: layout,
+  classNames: ['frost-action-link'],
+  classNameBindings: ['isSelected'],
+
+  initContext: Ember.on('didInitAttrs', function () {
+    this.set('_frostObjectDetails', this.nearestOfType(_frostObjectDetails))
+  }),
+
+  isSelected: Ember.computed('actionLink', 'routing.currentRouteName', function () {
```

Comment:
This component needs to act as a link (e.g. right click menu allows open in new tab, etc.) so I need confirmation that the current component supports this, or we need to make a change.